tests/test_diffeq/test_ode/test_ivp_examples.py

```diff
@@ -78,9 +78,26 @@ def test_threebody(self):
         self.assertIsInstance(lg2, ivp.IVP)
         self.assertIsInstance(lg3, ivp.IVP)
 
+    def test_lorenz(self):
+        """Test the Lorenz model ODE convenience function."""
+        rv = Constant(np.array([1.0, 1.0, 1.0]))
+        lg1 = ivp_examples.lorenz(self.tspan, rv)
+        lg2 = ivp_examples.lorenz(
+            self.tspan,
+            rv,
+            params=(
```

Comment:
You might want to set the default parameters as `float`s, but I think it will make no difference.